POSTINSTALL.md

```diff
@@ -31,21 +31,33 @@ The `npm run import` is for use with the official Firebase Extension [**Search w
 #### Run the script
 
 The import script uses several values from your installation of the extension:
-- Cloud Functions Location: Where do you want to deploy the functions created for this extension? You usually want a location close to your database. For help selecting a location, refer to the [location selection guide](https://firebase.google.com/docs/functions/locations).
 
-- Collection Path: What is the path to the collection that you want to index?
+- Cloud Functions Location: Where do you want to deploy the functions created for this extension?
+  You usually want a location close to your database.
+  For help selecting a location, refer to the
+  [location selection guide](https://firebase.google.com/docs/functions/locations).
 
-- Fields: What are fields in the document do you want to index?
+- Collection Path: What is the path to the Cloud Firestore collection where the extension should monitor for changes?
 
-**NOTE:** This configuration can be left empty if you want to index all the fields in this document.
+- Fields: What document fields should be indexed to provide the best search experience?
+  This can be comma separated list or left blank to index all fields.
+  Refer to [reducing record size](https://www.algolia.com/doc/guides/sending-and-managing-data/prepare-your-data/how-to/reducing-object-size/).
+  For performance reasons, [record size is limited](https://www.algolia.com/doc/guides/sending-and-managing-data/prepare-your-data/in-depth/index-and-records-size-and-usage-limitations/#record-size-limits).
 
-- Algolia Application ID: What is the Algolia application id that contains the index?
+- Algolia Application ID: What is the Algolia Application Id?
+  This is the application that contains one or more indexes.
+  Refer to [application id](https://www.algolia.com/doc/guides/sending-and-managing-data/send-and-update-your-data/how-to/importing-with-the-api/#application-id).
 
-- Algolia API Key: What is the Algolia API key with addObject, deleteObject, listIndexes, deleteIndex, editSettings, and settings permissions.
+- Algolia API Key: What is your Algolia API key?
+  We recommend creating a new [api key](https://www.algolia.com/doc/guides/security/api-keys/#creating-and-managing-api-keys)
+  with "addObject", "deleteObject", "listIndexes", "deleteIndex", "editSettings", and "settings" permissions.
+  **Do not use the Admin API key**.
 
-- Algolia Index Name: What is the Algolia index name that will contain the collection document records.
+- Algolia Index Name: What is the Algolia index name?
+  This is the name of the store where the records will be persisted.
+  Refer to [naming your index](https://www.algolia.com/doc/guides/sending-and-managing-data/send-and-update-your-data/#naming-your-index).
```

Comment:
IIUC, users will already have made this configuration (i.e., when they installed the extension) by the time they get to the point of running this script. If so, I don't think you need to list them here.

Reply:
Can you provide more clarity here?